files/templates/alertmanager/notification-template.tmpl

```diff
@@ -1,13 +1,23 @@
 {{ define "__alertmanager" }}Alertmanager{{ end }}
-{{ define "__alertmanagerurl" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}&silenced=false&inhibited=false&active=true&filter=%7Balertname%3D%22{{ .CommonLabels.alertname }}%22%7D{{ end }}
+
+{{ define "__alertmanagerurl" }}
+[[- if .MimirEnabled ]]
+[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find
+[[- else ]]
+{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}&silenced=false&inhibited=false&active=true&filter=%7Balertname%3D%22{{ .CommonLabels.alertname }}%22%7D
+[[- end ]]
+{{ end }}
+
 {{ define "__dashboardurl" -}}{{ if match "^https://.+" (index .Alerts 0).Annotations.dashboard }}{{ (index .Alerts 0).Annotations.dashboard }}{{ else }}[[ .GrafanaAddress ]]/d/{{ (index .Alerts 0).Annotations.dashboard }}{{ end }}{{- end }}
 {{ define "__runbookurl" -}}https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/{{ (index .Alerts 0).Annotations.opsrecipe }}{{- end }}
 
+{{ define "__queryurl" }}
 [[- if .MimirEnabled ]]
-{{ define "__queryurl" }}[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find{{end}}
+[[ .GrafanaAddress ]]/alerting/Mimir/{{ .CommonLabels.alertname }}/find
 [[- else ]]
-{{ define "__queryurl" }}{{ (index .Alerts 0).GeneratorURL }}{{end}}
+{{ (index .Alerts 0).GeneratorURL }}
 [[- end ]]
+{{ end }}
 
 {{ define "slack.default.title" }}{{ .Status | toUpper }}[{{ if eq .Status "firing" }}{{ .Alerts.Firing | len }}{{- else }}{{ .Alerts.Resolved | len }}{{- end }}] {{ (index .Alerts 0).Labels.alertname }} - Team {{ (index .Alerts 0).Labels.team }}{{ end }}
```

Comment:
I think you forgot the slack.default.titlelink link

Reply:
Well I set the grafana link in __alertmanagerurl variable in case of mimir.

Reply:
You're right but I think renaming the variable to alerturl makes more sense than alertmanagerurl?